pyam/core.py

```diff
@@ -1303,16 +1313,11 @@ def aggregate_region(
             weight=weight,
         )
 
-        # return None if there is nothing to aggregate
-        if _df is None:
-            return None
-
         # else, append to `self` or return as `IamDataFrame`
         if append is True:
```

Comment:
Why are you explicitly interpreting append=1 as append=False?

```suggestion
        if append:
```